Packages/Status/Sources/Status/Row/StatusCardView.swift

```diff
@@ -64,7 +64,7 @@ public struct StatusCardView: View {
         ShareLink(item: url) {
           Label("Share this link", systemImage: "square.and.arrow.up")
         }
-        Button { openURL(url) } label: {
+        Button { UIApplication.shared.open(url) } label: {
```

Comment:
But for the card, you still want to use the routeur right? It should respect the open in app browser vs open in system browser.

Reply:
Not sure where the card is located, I was following the "Open in browser" translation.
Problem is that currently, openURL called on a Mastodon url somehow always end up to be an in-app trigger from my understanding

Reply:
It's fine for card, card are the big link thing in some statuses. Basically it'll route in the app if it's a mastodon link and route to the correct browser if it's not. So it's fine to leave untouched there.

Reply:
Let it be just a *very* small cleanup PR then